README.md

```diff
@@ -36,9 +36,6 @@ Required for parallel execution of implicit time integration (quasi-statics).
 
 https://trilinos.org
 
-# VT
+## BVH
```

Comment:
Maybe we should have both? Since BVH is so heavily built on VT

Reply:
There is no option for compiling with VT right now. 
It does not fit the sentence
"NimbleSM may be built with the following optional dependencies:"

Reply:
@nmm0 

I have updated the paragraph. Let me know what you think if you have time.

Reply:
Awesome thanks!